src/screens/Notification.js

```diff
@@ -361,6 +361,22 @@ background: {
             <Example defaultValue>"background-front"</Example>
           </PropertyValue>
         </Property>
+        <Property name="notification.critical.message.color">
+          <Description>The color of the message.</Description>
```

Comment:
These should be more specific (e.g. The color of the message text for critical status.)